src/componentLibrary/InfoBox/InfoBox.tsx

```diff
@@ -0,0 +1,31 @@
+import { CTA } from '../CTA';
+
+import { InfoBoxWrapper, InfoBoxHeader, InfoBoxText } from './InfoBox.style';
+
+interface InfoBoxProps {
+  title: string;
+  text: string;
+  ctaText?: string;
+  ctaOnClick?: () => void;
+}
+
+const InfoBox = ({ title, text, ctaText, ctaOnClick }: InfoBoxProps): JSX.Element => {
+  return (
+    <InfoBoxWrapper>
+      <div>
+        <InfoBoxHeader>{title}</InfoBoxHeader>
+        <InfoBoxText>{text}</InfoBoxText>
+      </div>
+      {ctaOnClick && (
+        <div>
```

Comment:
L20: Is it necessary to wrap CTA in div element? It seems redundant to me.

Reply:
Good spot :) that was there to fix a layout issue that went away when the CSS was finished.